src/officehours_api/backends/zoom.py

```diff
@@ -167,9 +174,7 @@ def save_user_meeting(cls, backend_metadata: dict, assignee: User):
         if backend_metadata.get('meeting_id'):
             return backend_metadata
 
-        user_email = backend_metadata['user_email']
-        user = User.objects.get(email=user_email)
-        meeting = cls._create_meeting(user)
+        meeting = cls._create_meeting(assignee)
```

Comment:
It looks like this was causing the meeting to always be created for the first host, which would often not be the assignee, but in testing scenarios, often *would* be.

Reply:
Might be missing something, but this sounds serious enough that it should be documented in a separate issue. Doesn't need to be a separate PR, in my view, though.

Reply:
How would you reproduce this bug/case in QA?

Reply:
* Create a queue with two hosts
* Create a meeting and assign it to the second host
* Start the meeting
* Correct behavior: underlying Zoom meeting is owned by the second host. Bad behavior: Is owned by first host.

#283

Reply:
Awesome, thanks. I'll check to make sure it's working.

Reply:
I might have to wait to test this until it's in QA, unless you've pushed this to `dev`. I don't have access to two umich accounts to use locally.

Reply:
Pushed it to dev.